Library/Formula/gtk-mac-integration.rb

```diff
@@ -0,0 +1,162 @@
+require 'formula'
+
+class GtkMacIntegration < Formula
+  homepage 'https://wiki.gnome.org/Projects/GTK%2B/OSX'
+  url      'http://ftp.gnome.org/pub/gnome/sources/gtk-mac-integration/2.0/gtk-mac-integration-2.0.7.tar.xz'
+  sha256   'e6de70da740b452f18c113a7ebb6772936a0fb9873137514090849820e296eb7'
```

Comment:
Your indentation is off here. Space is too wide between ` url ` and the url, and so on.